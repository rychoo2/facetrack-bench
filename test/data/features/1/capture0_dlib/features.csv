filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_eye_distance_x,rel_eye_distance_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5427509293680297,0.6044776119402985,0.6617100371747212,-0.05970149253731343,0.46715328467153283,0.475,0.43283582089552236,0.4358974358974359
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5427509293680297,0.6044776119402985,0.6617100371747212,-0.05970149253731343,0.46715328467153283,0.475,0.43283582089552236,0.4358974358974359
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459716796875,0.2431785300925926,20191123163438000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5613382899628253,0.6007462686567164,0.6579925650557621,-0.05970149253731343,0.4925373134328358,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459052191840277,0.24323157793209876,20191123163438000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5613382899628253,0.6007462686567164,0.6579925650557621,-0.05970149253731343,0.4925373134328358,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.4701492537313433,0.6194029850746269,0.6417910447761194,-0.05970149253731343,0.4881889763779528,0.46153846153846156,0.4251968503937008,0.46153846153846156
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.4701492537313433,0.6194029850746269,0.6417910447761194,-0.05970149253731343,0.4881889763779528,0.46153846153846156,0.4251968503937008,0.46153846153846156
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.2961222330729167,0.9832947530864198,20191123163458000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.5279503105590062,0.5714285714285714,0.5496894409937888,-0.037267080745341616,,,0.48175182481751827,0.5
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.29605712890625,0.9832368827160494,20191123163458000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.5279503105590062,0.5714285714285714,0.5496894409937888,-0.037267080745341616,,,0.48175182481751827,0.5
images/20191123163503.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06577284071180556,0.07994550540123457,20191123163503000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5335820895522388,0.585820895522388,0.6194029850746269,-0.0037313432835820895,0.5950413223140496,0.15,0.4961832061068702,0.3953488372093023
images/20191123163502.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06580539279513889,0.07994550540123457,20191123163502000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5298507462686567,0.585820895522388,0.6343283582089553,-0.007462686567164179,0.5725806451612904,0.23255813953488372,0.5,0.46511627906976744
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9681871202256944,0.9810860339506173,20191123163500000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.48134328358208955,0.6231343283582089,0.6417910447761194,-0.055970149253731345,0.46153846153846156,0.43478260869565216,0.4015748031496063,0.5384615384615384
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9680894639756944,0.9810860339506173,20191123163500000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.48134328358208955,0.6231343283582089,0.6417910447761194,-0.055970149253731345,0.46153846153846156,0.43478260869565216,0.4015748031496063,0.5384615384615384
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3268853081597222,0.2927396797839506,20191123163448000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.593167701863354,0.6452513966480447,0.5434782608695652,-0.04748603351955307,0.5041322314049587,0.4,0.48951048951048953,0.46511627906976744
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32685275607638886,0.2927396797839506,20191123163448000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.593167701863354,0.6452513966480447,0.5434782608695652,-0.04748603351955307,0.5041322314049587,0.4,0.48951048951048953,0.46511627906976744
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3080810546875,0.1638503086419753,20191123163444000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5745341614906833,0.6312849162011173,0.5403726708074534,-0.04189944134078212,0.5041322314049587,0.4418604651162791,0.4857142857142857,0.4782608695652174
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.30804850260416666,0.16390576774691357,20191123163444000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5745341614906833,0.6312849162011173,0.5403726708074534,-0.04189944134078212,0.5041322314049587,0.4418604651162791,0.4857142857142857,0.4782608695652174
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5186567164179104,0.5932835820895522,0.6529850746268657,-0.041044776119402986,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5186567164179104,0.5932835820895522,0.6529850746268657,-0.041044776119402986,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5186567164179104,0.5932835820895522,0.6529850746268657,-0.041044776119402986,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5186567164179104,0.5932835820895522,0.6529850746268657,-0.041044776119402986,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997477213541667,0.8314983603395062,20191123163456000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.46273291925465837,0.5869565217391305,0.5652173913043478,-0.046583850931677016,0.48507462686567165,0.4666666666666667,,
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997151692708333,0.8314983603395062,20191123163456000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.46273291925465837,0.5869565217391305,0.5652173913043478,-0.046583850931677016,0.48507462686567165,0.4666666666666667,,
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.5714285714285714,0.577639751552795,0.562111801242236,-0.024844720496894408,,,0.5104895104895105,0.48717948717948717
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.5714285714285714,0.577639751552795,0.562111801242236,-0.024844720496894408,,,0.5104895104895105,0.48717948717948717
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39939643012152776,0.8062644675925926,20191123163441000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.5671641791044776,0.6343283582089553,0.664179104477612,-0.04477611940298507,0.5153846153846153,0.3333333333333333,0.48905109489051096,0.4666666666666667
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39936387803819445,0.8062644675925926,20191123163441000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.5671641791044776,0.6343283582089553,0.664179104477612,-0.04477611940298507,0.5153846153846153,0.3333333333333333,0.48905109489051096,0.4666666666666667
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.808227237654321,20191123163440000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5464684014869888,0.6156716417910447,0.6505576208178439,-0.055970149253731345,0.48854961832061067,0.4444444444444444,0.4351145038167939,0.5
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.8081693672839506,20191123163440000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5464684014869888,0.6156716417910447,0.6505576208178439,-0.055970149253731345,0.48854961832061067,0.4444444444444444,0.4351145038167939,0.5
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13788926866319445,0.2912181712962963,20191123163435000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5838509316770186,0.6508379888268156,0.5279503105590062,-0.04748603351955307,0.5526315789473685,0.3939393939393939,0.5074626865671642,0.4722222222222222
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13785671657986112,0.2912157600308642,20191123163435000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5838509316770186,0.6508379888268156,0.5279503105590062,-0.04748603351955307,0.5526315789473685,0.3939393939393939,0.5074626865671642,0.4722222222222222
images/20191123163436.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.4290378146701389,0.3449291087962963,20191123163436000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5261194029850746,0.6156716417910447,0.6380597014925373,-0.04477611940298507,0.5,0.35,0.44776119402985076,0.4883720930232558
images/20191123163437.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.42900526258680555,0.3449291087962963,20191123163437000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5223880597014925,0.6156716417910447,0.6380597014925373,-0.048507462686567165,0.49586776859504134,0.35,0.44029850746268656,0.5
