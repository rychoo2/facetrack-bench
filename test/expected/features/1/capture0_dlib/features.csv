filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5539033457249071,0.6194029850746269,0.5724907063197026,0.13805970149253732,0.10408921933085502,0.19029850746268656,0.7434944237918215,0.20149253731343283,0.6617100371747212,-0.05970149253731343,0.25650557620817843,0.23880597014925373,0.4423076923076923,0.5151515151515151,0.40594059405940597,0.4482758620689655
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459716796875,0.2431785300925926,20191123163438000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.458955223880597,0.6156716417910447,0.47761194029850745,0.1417910447761194,-0.0037313432835820895,0.1865671641791045,0.6567164179104478,0.19776119402985073,0.6716417910447762,-0.05970149253731343,0.16791044776119404,0.23880597014925373,0.46153846153846156,0.46875,0.4423076923076923,0.4375
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.47761194029850745,0.6529850746268657,0.5,0.208955223880597,0.014925373134328358,0.26119402985074625,0.6716417910447762,0.24253731343283583,0.667910447761194,-0.06343283582089553,0.17537313432835822,0.2873134328358209,0.45544554455445546,0.42857142857142855,0.41346153846153844,0.4117647058823529
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.2961222330729167,0.9832947530864198,20191123163458000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.5403726708074534,0.5652173913043478,0.531055900621118,0.19875776397515527,0.11801242236024845,0.2422360248447205,0.6801242236024845,0.2204968944099379,0.546583850931677,-0.055900621118012424,0.2732919254658385,0.2670807453416149,0.5161290322580645,0.4117647058823529,0.47619047619047616,0.47058823529411764
images/20191123163503.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06577284071180556,0.07994550540123457,20191123163503000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5485074626865671,0.585820895522388,0.5335820895522388,0.10820895522388059,0.029850746268656716,0.1417910447761194,0.75,0.16791044776119404,0.667910447761194,-0.04477611940298507,0.26119402985074625,0.208955223880597,0.5979381443298969,0.43243243243243246,0.5535714285714286,0.5
images/20191123163502.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06580539279513889,0.07994550540123457,20191123163502000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5447761194029851,0.5895522388059702,0.5298507462686567,0.10820895522388059,0.029850746268656716,0.1455223880597015,0.746268656716418,0.17164179104477612,0.667910447761194,-0.048507462686567165,0.2574626865671642,0.2126865671641791,0.5979381443298969,0.4594594594594595,0.5596330275229358,0.5
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9681871202256944,0.9810860339506173,20191123163500000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.48880597014925375,0.6417910447761194,0.503731343283582,0.20522388059701493,0.026119402985074626,0.2574626865671642,0.6716417910447762,0.23507462686567165,0.6716417910447762,-0.05970149253731343,0.1865671641791045,0.2835820895522388,0.4326923076923077,0.4,0.42574257425742573,0.4375
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3268853081597222,0.2927396797839506,20191123163448000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.6086956521739131,0.6368715083798883,0.5714285714285714,0.27932960893854747,0.16149068322981366,0.31843575418994413,0.717391304347826,0.31843575418994413,0.5372670807453416,-0.05307262569832402,0.3167701863354037,0.36033519553072624,0.5108695652173914,0.4375,0.45871559633027525,0.46875
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3080810546875,0.1638503086419753,20191123163444000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5838509316770186,0.6284916201117319,0.5590062111801242,0.2709497206703911,0.14285714285714285,0.30446927374301674,0.6987577639751553,0.3128491620111732,0.5372670807453416,-0.0446927374301676,0.2981366459627329,0.3463687150837989,0.5056179775280899,0.45454545454545453,0.46296296296296297,0.45454545454545453
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5298507462686567,0.5932835820895522,0.5186567164179104,0.12686567164179105,0.007462686567164179,0.16791044776119404,0.7014925373134329,0.17537313432835822,0.6753731343283582,-0.06343283582089553,0.2126865671641791,0.22388059701492538,0.5051546391752577,0.4482758620689655,0.49107142857142855,0.46875
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997477213541667,0.8314983603395062,20191123163456000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.46273291925465837,0.5900621118012422,0.484472049689441,0.20496894409937888,0.08074534161490683,0.2453416149068323,0.639751552795031,0.2360248447204969,0.5652173913043478,-0.043478260869565216,0.22981366459627328,0.2670807453416149,0.47619047619047616,0.4,0.45714285714285713,0.3333333333333333
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.4671875,0.0375,0.2515625,0.44722222222222224,0.593167701863354,0.577639751552795,0.577639751552795,0.20186335403726707,0.16149068322981366,0.2422360248447205,0.7329192546583851,0.2204968944099379,0.531055900621118,-0.059006211180124224,0.32919254658385094,0.2701863354037267,0.5882352941176471,0.3,0.5142857142857142,0.42857142857142855
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39939643012152776,0.8062644675925926,20191123163441000,0.45859375,0.03194444444444444,0.209375,0.37222222222222223,0.585820895522388,0.6492537313432836,0.5895522388059702,0.208955223880597,0.08582089552238806,0.2537313432835821,0.7649253731343284,0.23507462686567165,0.6716417910447762,-0.05970149253731343,0.27611940298507465,0.2873134328358209,0.4845360824742268,0.3333333333333333,0.4722222222222222,0.42857142857142855
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.808227237654321,20191123163440000,0.43515625,0.03194444444444444,0.21015625,0.37222222222222223,0.5576208178438662,0.6455223880597015,0.5762081784386617,0.19776119402985073,0.10408921933085502,0.24253731343283583,0.7546468401486989,0.23507462686567165,0.6617100371747212,-0.05223880597014925,0.26765799256505574,0.2798507462686567,0.46153846153846156,0.4166666666666667,0.4230769230769231,0.47619047619047616
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13788926866319445,0.2912181712962963,20191123163435000,0.4671875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5993788819875776,0.6452513966480447,0.5683229813664596,0.29608938547486036,0.15838509316770186,0.329608938547486,0.717391304347826,0.329608938547486,0.5248447204968945,-0.055865921787709494,0.3198757763975155,0.3715083798882682,0.5647058823529412,0.41379310344827586,0.5,0.46875
images/20191123163436.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.4290378146701389,0.3449291087962963,20191123163436000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5410447761194029,0.6194029850746269,0.5149253731343284,0.15298507462686567,0.026119402985074626,0.208955223880597,0.6865671641791045,0.20149253731343283,0.6567164179104478,-0.07462686567164178,0.20522388059701493,0.2574626865671642,0.4791666666666667,0.4642857142857143,0.45714285714285713,0.4642857142857143
images/20191123163437.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.42900526258680555,0.3449291087962963,20191123163437000,0.48203125,0.03194444444444444,0.209375,0.37222222222222223,0.5373134328358209,0.6194029850746269,0.5149253731343284,0.15298507462686567,0.014925373134328358,0.20522388059701493,0.6828358208955224,0.20149253731343283,0.664179104477612,-0.07462686567164178,0.20149253731343283,0.2574626865671642,0.46875,0.4482758620689655,0.46296296296296297,0.4827586206896552
